tests/ceil-double-varying.ispc

```diff
@@ -0,0 +1,72 @@
+// rule: skip on cpu=TGLLP
+
+export void f_f(uniform float RET[], uniform float aFOO[]) {
+    float error = 0;
+    RET[programIndex] = 0;
+    double deltaLow = 0.2d;
+    double deltaHigh = 0.8d;
+    double testVal;
+    double resVal;
+    double baseVal;
+    double checkVal = 0;
+
+    // Case 1: Covers range  -(programCount/2 -1) < val < +(programCount/2) including -1 < val < +1 with deltaLow.
+    // So, this covers +0.0, -0.0, -inf < val < 0 with decimal greater than 0.5, 0 < val < +inf with decimal less than 0.5
```

Comment:
This doesn't cover `+0.0` and `-0.0` as special cases. I.e. it doesn't test that `ceil(+0.0) == +0.0`. So they need to be checked separately.

Reply:
It's good to go now.